### PR TITLE
feat: add docs for receipt-link attribute

### PR DESCRIPTION
**Pull request by markusahlstrand**
Forgot to update the docs for this..

### CODE REVIEW

File: docs/news-widget/web-components/sesamy-content-container.md [hunk 1]
```diff
@@ -98,12 +98,18 @@ When using the `lock-mode=signedUrl` lock mode the content is fetched from the s
...
 
-### Unlock Message
+### Receipt Link
 
 This message is shown below the article once it is unlocked, it contains a link to the confirmation screen of the checkout where te item was purchased.
-It can be hidden by including the attribute `hide-unlock-message` in the sesamy-content-container tag (`<sesamy-content-container hide-unlock-message>`).
+It can be configured by setting the attribute `receipt-link` in the sesamy-content-container tag (`<sesamy-content-container receipt-link="none">`).
 
-The styling of the unlocke message can be configured with the following css variables:
+The following options are available fo the `receipt-link` attribute:
```
Comment (danielgent) [inline docs/news-widget/web-components/sesamy-content-container.md:106]: `foR`  :wink

Reads like https://www.urbandictionary.com/define.php?term=Fo%20sure

File: docs/news-widget/web-components/sesamy-content-container.md [hunk 2]
```diff
@@ -98,12 +98,18 @@ When using the `lock-mode=signedUrl` lock mode the content is fetched from the s
...
+
+- Text (default) (`receipt-link="text"`): Displays a `powered by Sesamy` message
+- Embed (`receipt-link="embed"`): Displays a receipt in a embedded window
```
Comment (danielgent) [inline docs/news-widget/web-components/sesamy-content-container.md:109]: `aN embedded window` (word starts with a vowel) :upside_down_face: 

Apologies for being pedantic but the docs are customer facing!

**Review by danielgent — CHANGES_REQUESTED** [review]
Two more typos!

File: docs/news-widget/web-components/sesamy-content-container.md (hunk 2 above)
Comment (markusahlstrand) [reply docs/news-widget/web-components/sesamy-content-container.md:109]: I actually incorrectly thought that `a` or `an` was decided by the noun and not the adjective :) In other languages you typically have a genus which won't change depending on the adjective. And in Swedish we have the same behaviour with `en` or `ett` but it's also defined by the noun. Interesting!

**Comment by danielgent** [comment]
@markusahlstrand you've got conflicts!

File: docs/news-widget/web-components/sesamy-content-container.md [hunk 3]
```diff
@@ -98,12 +98,18 @@ When using the `lock-mode=signedUrl` lock mode the content is fetched from the s
...
 This message is shown below the article once it is unlocked, it contains a link to the confirmation screen of the checkout where te item was purchased.
-It can be hidden by including the attribute `hide-unlock-message` in the sesamy-content-container tag (`<sesamy-content-container hide-unlock-message>`).
+It can be configured by setting the attribute `receipt-link` in the sesamy-content-container tag (`<sesamy-content-container receipt-link="none">`).
 
-The styling of the unlocke message can be configured with the following css variables:
+The following options are available for the `receipt-link` attribute:
+
+- Text (default) (`receipt-link="text"`): Displays a `powered by Sesamy` message
+- Embed (`receipt-link="embed"`): Displays a receipt in an embedded window
+- None (`receipt-link="none"`): Removed the receipt link message
```
Comment (danielgent) [inline docs/news-widget/web-components/sesamy-content-container.md:110]: Let's just merge this Markus but it shouldn't be the past-tense `removed` as everything is present-continous!  (I presume it should be `removes`

Thinking about it, Grammarly wouldn't know how to correct this anyway as they're not real sentences

**Comment by markusahlstrand** [comment]
Want me to fix the conflicts?

**Comment by danielgent** [comment]
> Ahh.. thought this was your PR.. I'll fix the conflicts!

image

**Review by danielgent — APPROVED** [review]
This bikeshed looks perfect!

**Pull request merged**